caikit/runtime/model_management/model_manager.py

```diff
@@ -148,8 +148,13 @@ def __init__(self):
 
         # Do the initial local models load
         if self._local_models_dir:
-            log.info("<RUN44739400I>", "Loading local models into Caikit Runtime...")
-            self.sync_local_models(wait=True)
+            wait = runtime_cfg.wait_for_initial_model_loads
+            log.info(
+                "<RUN44739400I>",
+                "Loading local models into Caikit Runtime. Wait: %s",
+                wait,
+            )
+            self.sync_local_models(wait=wait)
```

Comment:
should the `wait` parameter here be renamed to not be confused with the `wait` on `load_model` that waits for a single load, where this one waits for all loads to finish?

I'd lean no, but just bringing it up. We probably could update the docstring on `sync_local_models` to elaborate a bit that the model loads happen in parallel and the `wait` waits on all of them after queueing them all up

Reply:
Yeah, I think elaborating on the docstring is the right thing. I'll do that